### PR TITLE
ISSITES-5816 Styles for Public Navigation

### PR DESCRIPTION
**Pull request by liaoethan**

### CODE REVIEW

File: modules/apps/site-initializer/site-initializer-liferay-learn/src/main/resources/site-initializer/fragments/group/public-sites-navigation/fragments/public-sites-navigation/index.scss [hunk 1]
```diff
@@ -18,6 +18,18 @@ nav.public-sites-navigation {
 		max-width: 1366px;
 	}
 
+	.primary-nav {
+		background: var(--white);
```
Comment (jotchon) [inline modules/apps/site-initializer/site-initializer-liferay-learn/src/main/resources/site-initializer/fragments/group/public-sites-navigation/fragments/public-sites-navigation/index.scss:26]: Is there a reason this is set to a white background?  Shouldn't the background be set with the dark-nav?

**Comment by liaoethan** [comment]
Hi @jotchon, thanks for the review! I made the changes that you specified except for the dropdown's being open at the same time. That will be done in the DropdownProvider ticket: https://issues.liferay.com/browse/ISSITES-5784

File: modules/apps/site-initializer/site-initializer-liferay-learn/src/main/resources/site-initializer/fragments/group/public-sites-navigation/fragments/public-sites-navigation/index.scss [hunk 2]
```diff
@@ -304,6 +332,10 @@ nav.public-sites-navigation {
 				visibility: visible;
 				z-index: 2;
 			}
+
+			.adt-nav-text {
```
Comment (jotchon) [inline modules/apps/site-initializer/site-initializer-liferay-learn/src/main/resources/site-initializer/fragments/group/public-sites-navigation/fragments/public-sites-navigation/index.scss]: Is there a better way to add this padding?  Ideally we don't want to use '!important'.

File: modules/apps/site-initializer/site-initializer-liferay-learn/src/main/resources/site-initializer/style-books/learn/frontend-tokens-values.json [hunk 3]
```diff
@@ -54,6 +54,14 @@
 		"cssVariableMapping": "color-neutral-10",
 		"value": "#282934"
 	},
+	"colorNeutral2": {
+		"cssVariableMapping": "color-neutral-2",
+		"value": "#E2E2E4"
```
Comment (jotchon) [inline modules/apps/site-initializer/site-initializer-liferay-learn/src/main/resources/site-initializer/style-books/learn/frontend-tokens-values.json:59]: Could you double check these hex values?  Neutral 2 should be #394452 and Neutral 8 should be ##ebeef2

**Comment by jotchon** [comment]
@liaoethan Let me know if you have any questions! (I also made some notes on the ticket as well!)

File: modules/apps/site-initializer/site-initializer-liferay-learn/src/main/resources/site-initializer/style-books/learn/frontend-tokens-values.json (hunk 3 above)
Comment (liaoethan) [reply modules/apps/site-initializer/site-initializer-liferay-learn/src/main/resources/site-initializer/style-books/learn/frontend-tokens-values.json:59]: In figma, it says that neutral 8 is #54555F
image

neutral 2 is #E2E2E4
image

**Comment by liaoethan** [comment]
Hi @jotchon, thanks for all of the feedback! I made the changes that you specified, let me know if that looks good. Thanks!

**Comment by jotchon** [comment]
Hi @liaoethan, 

We discussed that the user account dropdown arrow interaction arrow should point up when dropdown is open.
Screen Shot 2022-11-28 at 1 14 12 PM

Please create a new PR for this and also rebase.  Thank you!

Other than that, it looks great!  Awesome job! :) 

Johanna

**Comment by liaoethan** [comment]
Thanks @jotchon, I made the change here: https://github.com/allen-ziegenfus/liferay-portal/pull/33

**Pull request closed without merging**